BangazonAPI/BangazonAPI/Models/Employee.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace BangazonAPI.Models
+{
+    public class Employee
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        public int DepartmentId { get; set; }
+        public bool IsSupervisor { get; set; }
+    
```

Comment:
According to ticket #6 a representation of the computer that the employee is currently using should be included in the employee representation".

Reply:
🍝

Reply:
-I added an instance of computer as a property to employee.
-Also added an empty class called computer.